test/jdk/javax/swing/SwingWorker/TestDoneBeforeDoInBackground.java

```diff
@@ -86,9 +122,10 @@ protected void done() {
         if (!doneLatch.await(CLEANUP_TIME + 1000L, TimeUnit.MILLISECONDS)) {
             throw new RuntimeException("done didn't complete in time");
         }
-        System.out.println("doInBackground " + doInBackground +
+        System.out.println("doInBackground " + doInBackgroundFinished.get() +
                            " getState " + worker.getState());
-        if (doInBackground && worker.getState() != SwingWorker.StateValue.DONE) {
+        if (doInBackgroundFinished.get() &&
+              worker.getState() != SwingWorker.StateValue.DONE) {
```

Comment:
```suggestion
        if (doInBackgroundFinished.get() &&
                worker.getState() != SwingWorker.StateValue.DONE) {
```
If you follow the style of double indentation on continuation lines, two more spaces should be added.

Consider moving the `&&` operator to the continuation line.